packages/component/src/Styles/StyleSet/SendBox.js

```diff
@@ -13,7 +13,7 @@ export default function createSendBoxStyle({
       borderLeft: sendBoxBorderLeft,
       borderRight: sendBoxBorderRight,
       borderTop: sendBoxBorderTop,
-      height: sendBoxHeight
+      minHeight: sendBoxHeight
```

Comment:
do you  need to add visual testing to make sure the input doesn't grow in size if the text starts scrolling or something?

Reply:
In my test, I made the text long enough that the textarea would have to scroll. Do you think I should add another test?

Reply:
If I set the height of the main div, then it won't grow as the height of it's children increase, which is why I used minHeight instead.